imager_images/views.py

```diff
@@ -1,3 +1,99 @@
 from django.shortcuts import render
+from imager_images.forms import UserForm, ImagerProfileForm
+# from media import profile_image
+from django.core.urlresolvers import reverse_lazy, reverse
+from models import Photo, Album
+from django.views.generic import CreateView, UpdateView
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.views import redirect_to_login
+from django.shortcuts import redirect
+from dimager.models import ImagerProfile
 
-# Create your views here.
+
+def register(request):
+    context = RequestContext(request)
+    registered = False
+
+    if request.method == 'POST':
+        user_form = UserForm(data=request.POST)
+        profile_form = ImagerProfileForm(data=request.POST)
+
+        if user_form.is_valid() and profile_form.is_valid():
+            user = user_form.save()
+            user.set_password(user.password)
+            user.save()
+            profile = profile_form.save(commit=False)
+            profile.user = user
+
+            if 'picture' in request.FILES:
+                profile.profile_image = request.FILES
+                profile.profile_image = request.FILES['picture']
+
+            profile.save()
+            registered = True
+
+        else:
+            print user_form.errors, profile_form.errors
```

Comment:
please don't leave debugging print statements in submitted code.  In fact, don't use print statements to debug.  Use logging instead.